wdl2cwl/main_oop.py

```diff
@@ -250,46 +252,61 @@ def get_cwl_inputs(
         """Convert WDL inputs into CWL inputs and return a list of CWL Command Input Paramenters."""
         inputs: List[cwl.CommandInputParameter] = []
 
-        if not wdl_inputs:
-            return inputs
-        for wdl_input in wdl_inputs:
```

Comment:
Can you keep this simplification? It is nice to not have to indent an extra level